csharp/autobuilder/Semmle.Autobuild.Shared/BuildScript.cs

```diff
@@ -70,7 +70,7 @@ public BuildCommand(string exe, string? argumentsOpt, bool silent, string? worki
                 this.environment = environment;
             }
 
-            public override string ToString() => exe + " " + arguments;
+            public override string ToString() => arguments.Length > 0 ? exe + " " + arguments : exe;
```

Comment:
Maybe use !string.IsNullOrEmpty.

Reply:
`arguments` cannot be `null` (we are using C# 8 nullable reference types), so we only need the emptiness check.

Reply:
Yes, let's keep it as it is :-)